### PR TITLE
Feature/tg 144 obtain meeting report

### PR DESCRIPTION
**Pull request by CrisCastroI**
# Summary

Added endpoint and services to generate a meeting report from webex

# Details

* Added meeting report and participant report DTO
* Added create report endpoint on meetings controller
* Save report information to MeetingAttendance table
* Added neccesary repository for MeetingAttendance
* Added neccesary method for report generation in meetings controller and repository

# Evidence

image

image

### CODE REVIEW

File: KlockanAPI/src/KlockanAPI.Domain/Models/Webex/MeetingReport.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KlockanAPI.Domain.Models.Webex
+{
+    public class MeetingReport
+    {
+        public List<ParticipantReport> items { get; set; }
+    }
+}
```
Comment (AJN-jalaU) [inline KlockanAPI/src/KlockanAPI.Domain/Models/Webex/MeetingReport.cs]: please change to 
`
namespace KlockanAPI.Domain.Models.Webex;

public class MeetingReport
{
     public List<ParticipantReport> items { get; set; }
}
`

File: KlockanAPI/src/KlockanAPI.Presentation/appsettings.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "ConnectionStrings": {
-    "DefaultConnection": "Host=ec2-13-59-173-197.us-east-2.compute.amazonaws.com;Port=5432;Database=klockandb;Username=postgres;Password=postgres"
-  },
+    "ConnectionStrings": {
+        "DefaultConnection": "Host=localhost;Port=5432;Database=klockandb;Username=postgres;Password=postgres"
+    },
   "Logging": {
     "LogLevel": {
```
Comment (AJN-jalaU) [inline KlockanAPI/src/KlockanAPI.Presentation/appsettings.json]: here should we keep the instance url

File: KlockanAPI/src/KlockanAPI.Domain/Models/Webex/ParticipantReport.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KlockanAPI.Domain.Models.Webex
+{
+    public class ParticipantReport
+    {
+        public string id { get; set; }
+        public bool host { get; set; }
```
Comment (AJN-jalaU) [inline KlockanAPI/src/KlockanAPI.Domain/Models/Webex/ParticipantReport.cs]: same here

File: KlockanAPI/src/KlockanAPI.Application/DTOs/Meeting/MeetingReportDTO.cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KlockanAPI.Application.DTOs.Meeting
```
Comment (AJN-jalaU) [inline KlockanAPI/src/KlockanAPI.Application/DTOs/Meeting/MeetingReportDTO.cs]: same here

**Review by AJN-jalaU — APPROVED** [review]

**Review by Adrian-Mendoza-Jala — APPROVED** [review]

**Pull request merged**